lib/tasks/load_rubric_field_items.rake

```diff
@@ -0,0 +1,22 @@
+require "csv"
+
+task :load_rubric_field_items, [:assignment] => :environment do |t, args|
+  rubric_fields = RubricField.where(:assignment=>Assignment.find(args[:assignment])).order(:id)
+  position = 1
+  CSV.foreach(File.expand_path("../rubric_field_items_#{args[:assignment]}.csv", __FILE__)) do |row|
+    field_name = row[0].strip
+    item_name = row[1].strip
+    point = row[2].strip
+
+    field = rubric_fields.find_by(:name=>field_name)
+    field = rubric_fields.create(:name=>field_name) if field.nil?
```

Comment:
If the rubric field does not exist, here we will create a rubric field that does not belongs to any assignment. I would suggest we just do the following insertion ops if the field exists, otherwise just do nothing.